docs/developers/coordinator.md

```diff
@@ -0,0 +1,610 @@
+# Hermez Node
+This tutorial describes how to launch a Hermez node. It starts by explaining how to launch a Boot Coordinator in localhost.
+Next, it describes how to initialize a Proof Server and how to connect it to the Boot Coordinator.
+The next section describes how to spin up a second Hermez node in synchronizer mode to track the rollup status independently from the Boot Coordinator. This second node will be launched in Rinkeby testnet.
+The last part of the tutorial includes an explanation on how to add a second Coordinator node to Hermez testnet that bids for the right to forge batches.
+ 
+1. [Preparing the Environment](#preparing-the-environment) 
+2. [Launching the Boot Coordinator](#launching-the-boot-coordinator)
+3. [Launching a Proof Server](#launching-a-proof-server)
+4. [Launching a Synchronizer Node](#launching-a-synchronizer-node)
+5. [Launching a Second Coordinator](#launching-a-second-coordinator-node)
+
+## Preparing the Environment
+Hermez node requires a PostgreSQL database and connectivity to an Ethereum node. In this part, we describe how you can set this environment
+up using docker containers.
+
+### Dependencies
```

Comment:
It could be useful to add the following links:
- install golang: https://golang.org/doc/install
- install docker: https://docs.docker.com/engine/install/ubuntu/
- install docker-compose: https://docs.docker.com/compose/install/